src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
 public class Application {
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0;i < 3;i++) {
```

Comment:
format your files. You can use IDEA's autoformatter - Ctrl Alt L for windows machines. There always should be a comma after a semicolon ; 
```suggestion
        for (int i = 0; i < 3; i++) {
```